src/gobstuf/stuf/brp/response_mapping.py

```diff
@@ -9,6 +9,8 @@ class Mapping(ABC):
 
     Provides a filter method to filter out attributes and/or objects
     """
+    related = {}
+    is_relation = False
 
```

Comment:
Do all Mappings share the same related and is_relation?
It is now a class variable that is shared by all subclasses. I don't know if this is the intention